### PR TITLE
Refactor company export edit pages

### PR DESCRIPTION
**Pull request by cgsunkel**
## Description of change

I noticed that the `LocalHeaders` for the company Edit Export and Edit Export Countries forms were set in Nunjucks, so I moved these into React. In the process of doing this I refactored these two forms to get all data via the `Resource` component and cleared out some dead code related to the old submission mechanism.

This PR is not as big as it looks - most of the diff is from moving files to the modules directory.

## Test instructions

The two forms should work as normal

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/index.jsx [hunk 1]
```diff
@@ -34,8 +34,8 @@ import CompanyOverview from '../apps/companies/apps/company-overview/client/Comp
 import EditOneListForm from '../apps/companies/apps/edit-one-list/client/EditOneListForm'
 import ExportsIndex from '../apps/companies/apps/exports/client/ExportsIndex'
 import ExportsHistory from '../apps/companies/apps/exports/client/ExportsHistory/'
-import ExportsEdit from '../apps/companies/apps/exports/client/ExportsEdit.jsx'
-import ExportCountriesEdit from '../apps/companies/apps/exports/client/ExportCountriesEdit/'
+import ExportsEdit from './modules/Companies/CompanyExports/ExportsEdit'
+import ExportCountriesEdit from './modules/Companies/CompanyExports/ExportCountriesEdit/index'
```
Comment (paulgain) [inline src/client/index.jsx:38]: No need for the `/index`

File: src/client/modules/Companies/CompanyExports/ExportCountriesEdit/index.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          </StyledH2>
+          <StyledP>{errorMessage[API_WARN]}</StyledP>
+        </StatusMessage>
+      )}
+    </>
+  )
+}
+
+export default ({ companyId }) => {
+  return (
```
Comment (paulgain) [inline src/client/modules/Companies/CompanyExports/ExportCountriesEdit/index.jsx:59]: Favour implicit returns over explicit returns.

File: src/client/modules/Companies/CompanyExports/ExportsEdit.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled from 'styled-components'
+import { SPACING_POINTS } from '@govuk-react/constants'
+import { FieldInput, FieldSelect, LocalHeader, Main } from '../../../components'
```
Comment (paulgain) [inline src/client/modules/Companies/CompanyExports/ExportsEdit.jsx:4]: This should be with the other local imports below.

**Review by paulgain — APPROVED** [review]
Looks good, just a few minor comments.

File: src/client/modules/Companies/CompanyExports/ExportCountriesEdit/index.jsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        </StatusMessage>
+      )}
+    </>
+  )
+}
+
+export default ({ companyId }) => (
+  <CompanyResource id={companyId}>
+    {(company) => (
+      <CountriesResource>
```
[the remark this answers is not in the document]
Comment (cgsunkel) [reply src/client/modules/Companies/CompanyExports/ExportCountriesEdit/index.jsx:61]: I couldn't get it to work in this case as these fields are generated dynamically (which this field doesn't appear to support)

**Pull request merged**